<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDialog_q</class>
 <widget class="QWidget" name="TimeDialog_q">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame7">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="object_attributes_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Sunken</enum>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="obj_name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>200</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer>
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="object_attributes_2">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>16</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Sunken</enum>
        </property>
        <layout class="QVBoxLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox2">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout">
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel1_3">
              <property name="text">
               <string>Start time:</string>
              </property>
              <property name="toolTip">
               <string>Start of the daily time window when the rule is active (HH:MM, 24-hour format)</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="startTime">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Start of the daily time window when the rule is active (HH:MM, 24-hour format)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>111</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox1">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout">
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel1_3_2">
              <property name="text">
               <string>End time:</string>
              </property>
              <property name="toolTip">
               <string>End of the daily time window when the rule is active (HH:MM, 24-hour format)</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="endTime">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="toolTip">
               <string>End of the daily time window when the rule is active (HH:MM, 24-hour format)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>101</width>
                <height>25</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QGridLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QCheckBox" name="cbStart1_2">
                 <property name="text">
                  <string>Mon</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Mondays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="cbStart2_2">
                 <property name="text">
                  <string>Tue</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Tuesdays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QCheckBox" name="cbStart3_2">
                 <property name="text">
                  <string>Wed</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Wednesdays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QCheckBox" name="cbStart4_2">
                 <property name="text">
                  <string>Thu</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Thursdays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QCheckBox" name="cbStart5_2">
                 <property name="text">
                  <string>Fri</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Fridays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QCheckBox" name="cbStart6_2">
                 <property name="text">
                  <string>Sat</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Saturdays</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QCheckBox" name="cbStart7_2">
                 <property name="text">
                  <string>Sun</string>
                 </property>
                 <property name="toolTip">
                  <string>Enable this rule on Sundays</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CommentKeywords" name="commentKeywords" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommentKeywords</class>
   <extends>QWidget</extends>

   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>obj_name</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>endTime</tabstop>
 </tabstops>
 <resources>
  <include location="MainRes.qrc"/>
 </resources>
 <connections/>
</ui>
